plugins/ua_config_standard.c

```diff
@@ -9,6 +9,7 @@
 #include "ua_securitypolicy_none.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "ua_client_highlevel.h"
```

Comment:
No. This struct is only defined in the client file and is used in line 305 in the default config, since it supplies config for both server and client.